src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/order.yml

```diff
@@ -10,6 +10,15 @@ sylius_admin_api_order:
         serialization_version: $version
     type: sylius.resource_api
 
+sylius_admin_api_orders:
+    resource: |
+        alias: sylius.order
+        section: admin_api
+        only: [index]
```

Comment:
Can you merge this configuration with the configuration above and change `only: [show]` to `only: [show, index]`?